exercises/javascript-light-bulb/main.js

```diff
@@ -0,0 +1,15 @@
+var $lightBulb = document.querySelector('.light-bulb-on');
+var $row = document.querySelector('.row-on');
+var count = 0;
+function turnOnOff(event) {
+  if (count % 2 === 0) {
```

Comment:
Clever solution, but the count here isn't doing anything beyond being checked if it is even or odd, and we have another data type that lends itself really well to "this or that" situations: Booleans. I would make a variable named something like `isOn` set to `false` and toggle the value to the other boolean every time the button is clicked